DakeAndDevileCorps/Screens/Search/VC/RecentSearchTableViewCell.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  RecentSearchTableViewCell.swift
+//  DakeAndDevileCorps
+//
+//  Created by Seungyun Kim on 2022/07/19.
+//
+
+import UIKit
+
+class RecentSearchTableViewCell: UITableViewCell {
+
+    @IBOutlet weak var searchedTitle: UILabel!
+    
+    static let identifier = "RecentSearchTableViewCell"
```

Comment:
p5;

```suggestion
    static let identifier = "RecentSearchTableViewCell"
    
    @IBOutlet weak var searchedTitle: UILabel!
```

static 변수 중에서 identifier는 밖에서 접근하는 경우가 많기에 맨 윗쪽에다가 올려서 쓰면 읽기가 편하더라구요. 
제가 `identifier`를 안써도 되는 className이라는 친구를 곧 PR날릴 예정이라서 나중에 pull받으시면 사용해주시면 감사드리겠습니다!

Reply:
아하 그렇군요 감사합니다!!
혹시 그러면 그 외의 static 변수들도 가장 상단에 두는 게 좋을까요?

Reply:
[37ef4cc](https://github.com/DeveloperAcademy-POSTECH/MC3-Team4-DakeAndDevileCorps/pull/19/commits/37ef4cc35dc9f8d5901aa6b431d3ed346b16f270) 수정했습니다!